vcmi.spec

```diff
@@ -88,7 +88,7 @@ sed -i 's/GITDIR-NOTFOUND/%{version}/' cmake_modules/*
   -DCMAKE_INSTALL_RPATH_USE_LINK_PATH=ON \
   -DCMAKE_INSTALL_RPATH=%{_libdir}/%{name}
 
-%ifnarch %{ix86} x86_64 aarch64
+%ifnarch %{ix86} x86_64 aarch64 ppc64le
```

Comment:
Why add ppc64le to parallel builds, the builders haven't got the RAM to do it.

Reply:
@leigh123linux Sorry I am not aware of that. Let me remove that pp64le from matrix